app/lib/slack.js

```diff
@@ -41,9 +41,16 @@ module.exports = {
       slackOptions.channel = options.channel;
     }
 
-    return new Slack(options.webhookUrl || config.slack.hookUrl, {})
-    .send(slackOptions, (err) => {
-      console.error(err);
+    return new Promise((resolve, reject) => {
```

Comment:
I think `report_transactions.js` is the only place where it looks at the result of the call. All other places call the library blindly. 

But I'll add a test anyway.